index.html

```diff
@@ -4,12 +4,151 @@
+            <h1 class=" h1demo">ART & MUSIC<br>
+                FESTIVAL<br>
+                2022</h1>
+            <div class="p-2"><p class="p-2">A joyful manifestation in the value of opennes and sharing, 
+                creating a positive energy with performers from over 80 countries is taking place in July 07-11th, Novi Sad, Serbia</p>
+            <p class="p-2 large">2022.07.07.(THU) ~ 11(MON)</p>
+            <p class="p-2">@ All tickets for the Festival are individually personalised to the named ticket holder and are strictly non-transferable. Security checks are carried out on arrival, and only the specified ticket holder will be admitted to the Festival.</p>
+            </div>
+        </div>
+        <!--<img class="bg-img d-none d-sm-block" width="100%" src="./images/mainbg.png" alt="background"> -->   
+    </section>
+    <section class="bck container-fluid text-center text-white has-bg-img" id="programs">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12 text-center"><br>
+                    <h2 class="section-heading text-white">Programs</h2>
+            <hr class="my-4">
+        </div>
+    </div>
+        <div class="card-group my-list p-4 gap-2">
+        <div class="card card-background p-2 border border-danger bg-secondary bg-opacity-25">
+            <img class="card-img-top icon d-block" src="./images/icon1.png" alt="icon">
+        <div class="card-body">
+            <h4 class="card-tittle text-danger">Lecture</h4>
+            <p class="card-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+        </div>
+    </div>
+    <div class="card card-background p-1 border border-danger bg-secondary bg-opacity-25">
+        <img class="card-img-top icon" src="./images/icon2.png" alt="icon">
+    <div class="card-body">
+        <h4 class="card-tittle text-danger">Exibition</h4>
+        <p class="card-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+    </div>
+</div>
+<div class="card card-background border border-danger bg-secondary bg-opacity-25">
+    <img class="card-img-top icon" src="./images/icon3.png" alt="icon">
+<div class="card-body">
+    <h4 class="card-tittle text-danger">Forum</h4>
+    <p class="card-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+</div>
+</div>
+<div class="card card-background border border-danger bg-secondary bg-opacity-25">
+    <img class="card-img-top icon" src="./images/icon4.png" alt="icon">
+<div class="card-body">
+    <h4 class="card-tittle text-danger">Workshop</h4>
+    <p class="card-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+</div>
+</div>
```

Comment:
- [ ] It would be nice if you place a dark transparent background image on this section that closely looks like the Behance design. It doesn't look nice as empty as it is. 
- [ ] Please make the cards be as transparent as the design in Behance.
- [ ] Please add the title of this section and the bottom underline text just as shown in the design given. In your project, they are both missing. Kindly add them.

**The Behance Design**
![image](https://user-images.githubusercontent.com/50941074/164947409-6be813c7-bfbb-4026-b7d5-d43ae8223dfb.png)


** Your design**
![image](https://user-images.githubusercontent.com/50941074/164947394-dab87f62-12fe-49b0-b741-f3b8c340c5d7.png)

Reply:
Looks like something doesn't show on your side. Kindley check. Also, having a problem adding live view because I havent merge anything jet. that's why it's on the initial commit still.

Reply:
I cloned the repo and checked it, but nothing shows. Don't worry, try to fix them up and commit your changes then request for a review okay. I'm sure you have few more time.